primeqa/qg/run_qg.py

```diff
@@ -184,23 +184,42 @@ def main(raw_args):
 
     qg_model = QGModel(model_args.model_name_or_path, modality=model_args.modality)
 
+
+
     qgdl = QGDataLoader(
         tokenizer=qg_model.tokenizer,
         dataset_name=data_args.dataset_name,
         modality=model_args.modality,
         input_max_len=data_args.max_len,
         target_max_len=data_args.target_max_len,
     )
-    if training_args.do_train or training_args.do_eval:
-        train_dataset = qgdl.create(
-            dataset_split="train", dataset_config=data_args.dataset_config
-        )
-        valid_dataset = qgdl.create(
-            dataset_split="validation", dataset_config=data_args.dataset_config
-        )
 
-        compute_metrics = rouge_metrics(qg_model.tokenizer)
+    train_dataset = None
+    valid_dataset = None
+    if data_args.train_file is not None:
```

Comment:
Should this be: 
`if training_args.do_train and data_args.train_file is not None:`?